build.sh

```diff
@@ -362,11 +366,19 @@ fi
 
 # Disable the native build when targeting Windows.
 
-if [ "$__BuildOS" != "$__HostOS" ]; then
+if [ "$__BuildOS" != "$__HostOS" ] && [ $__CrossBuild == 0 ]; then
```

Comment:
Rereading it again, I think that I should have left things as they are for the condition, but changed the warning message to be more explicit. It should say something along the lines of:
 echo "Warning: compiling native components for a different OS is not yet supported"

Would you agree?